### PR TITLE
document default login

### PR DESCRIPTION
**Pull request by jflinchbaugh**
IBM contacts suggested documenting the default logins to assist in initial setup and test.

Build: https://jenkins.ci.sonatype.dev/job/integrations/job/cloud/job/operator-iq/job/feature-snapshots/job/document-default-logins/

### CODE REVIEW

File: scripts/templates/nxiq-operator-certified.vX.X.X-X.clusterserviceversion.yaml [hunk 1]
```diff
@@ -141,6 +141,8 @@ spec:
     Once the server instance is created by the operator and running,
     you'll want to expose the service as you see fit:
     1. Create a Route to the new service for iq.applicationPort (8070).
+    2. Visit the URL provided by the Route, and login with the default credentials:
+      `admin`/`admin123`
```
Comment (collinpeters) [inline scripts/templates/nxiq-operator-certified.vX.X.X-X.clusterserviceversion.yaml:145]: Are the backticks formatting? Wondering if this ends up being markdown or what and mainly if the README should have backticks as well. No biggie though.

**Review by collinpeters — APPROVED** [review]
+1

File: scripts/templates/nxiq-operator-certified.vX.X.X-X.clusterserviceversion.yaml (hunk 1 above)
Comment (jflinchbaugh) [reply scripts/templates/nxiq-operator-certified.vX.X.X-X.clusterserviceversion.yaml:145]: good point on readme. fixed it. if you expand this block to see following lines, you can see the table full of back-tick formatting that's valid in this description.

File: README.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ and push your operator image to quay.io to make it available for installation.
 8. Expose the new IQ Server outside the cluster: 
    2. Create a Route in OpenShift UI to the new service, port 8070.
 9. Visit the new URL shown on the Route page in OpenShift UI.
+10. Default credentials are `admin`/`admin123`.
```
Comment (kellyrob99) [inline README.md:24]: Should we include a note here to remind people to update these values?

**Review by kellyrob99 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jflinchbaugh) [reply README.md:24]: upon login, it'll remind you, but i can mention it here as well, in case they don't recognize the impact of the default logins.

Comment (jflinchbaugh) [reply README.md:24]: This is a doc for dev/testing, so I've not bothered mentioning it here, but I have added a note about setting credentials over in the description/docs that are shown in the operator hub that users would see.

**Pull request merged**